scripts/version updates/update_meilisearch_version.sh

```diff
@@ -0,0 +1,278 @@
+#!/bin/bash
+
+## Constants
+
+RED="\033[0;31m"
+BRED="\033[1;31m"
+BGREEN="\033[1;32m"
+BYELLOW="\033[1;33m"
+BBLUE="\033[1;34m"
+BPINK="\033[1;35m"
+WHITE="\033[0m"
+NC="\033[0m"
+ERROR_LABEL="${BRED}error: ${NC}"
+SUCCESS_LABEL="${BGREEN}success: ${NC}"
+PENDING_LABEL="${BYELLOW}pending: ${NC}"
+INFO_LABEL="${BBLUE}info: ${NC}"
+
+## Utils Functions
+
+# Rollback to previous MeiliSearch version in case something went wrong
+
+previous_version_rollback() {
+
+    echo "${ERROR_LABEL}MeiliSearch update to $meilisearch_version failed." >&2
+    echo "${INFO_LABEL}Rollbacking to previous version ${BPINK}$current_meilisearch_version${NC}." >&2
+    echo "${INFO_LABEL}Recovering..." >&2
+    mv /tmp/meilisearch /usr/bin/meilisearch
+    echo "${SUCCESS_LABEL}Recover previous data.ms." >&2
+    mv /tmp/data.ms /var/lib/meilisearch/data.ms
+    echo "${INFO_LABEL}Restarting MeiliSearch." >&2
+    systemctl restart meilisearch
+    systemctl_status exit
+    echo "${SUCCESS_LABEL}Previous MeiliSearch version ${BPINK}$current_meilisearch_version${NC} restarted correctly with its data recovered." >&2
+    delete_temporary_files
+    echo "${ERROR_LABEL}Update MeiliSearch from ${BPINK}$current_meilisearch_version${NC} to ${BPINK}$meilisearch_version${NC} failed. Rollback to previous version successfull." >&2
+    exit
+}
+
+# Check if MeiliSearch systemctl process is Active
+systemctl_status() {
+    systemctl status meilisearch | grep -E 'Active: active \(running\)' -q
+    grep_status_code=$?
+    callback_1=$1
+    callback_2=$2
+    if [ $grep_status_code -ne 0 ]; then
+        echo "${ERROR_LABEL}MeiliSearch Service is not Running. Please start MeiliSearch." >&2
+        if [ ! -z "$callback_1" ]; then
+            $callback_1
+        fi
+
+        if [ ! -z "$callback_2" ]; then
+            2
```

Comment:
I should have missed something, but what is the purpose of the `2` here?

Reply:
None! 
```suggestion
```